marblerun/workflows/verification.md

```diff
@@ -6,33 +6,33 @@ MarbleRun provides a simple HTTP REST API for clients to verify the confidential
 
 MarbleRun exposes the `/quote` endpoint that returns a quote and a certificate chain consisting of a root and intermediate CA. The root CA is fixed for the lifetime of your deployment, while the intermediate CA changes in case you [update](workflows/update-manifest.md) the packages specified in your manifest.
 
-The simplest way to verify the quote is via the Edgeless Remote Attestation ([era](https://github.com/edgelesssys/era)) tools:
+The simplest way to verify the quote is via the Edgeless Remote Attestation ([era](https://github.com/edgelesssys/era)) tool.
 
-```bash
-# Install era globally on your machine (requires root permissions)
-sudo wget -O /usr/local/bin/era https://github.com/edgelesssys/era/releases/latest/download/era
-sudo chmod +x /usr/local/bin/era
+To verify the coordinator, Era requires the Coordinator's UniqueID (or MRENCLAVE in SGX terms) or the tuple ProductID, SecurityVersion, SignerID (MRSIGNER) to verify the quote. Era contacts the Coordinator, and receives an SGX quote from it which contains the actual UniqueID or ProductID/SecurityVersion/SignerID tuple of the running instance. The tool verifies it against the values the expected values defined in `coordinator-era.json` and can therefore determine if an authentic copy of the Coordinator is running, or if an unknown version is contacted. 
 
-# Run era. You can remove the output parameters you do not need for your use case.
-era -c coordinator-era.json -h $MARBLERUN -output-chain marblerun-chain.pem -output-root marblerun-root.pem -output-intermediate marblerun-intermedite.pem
-```
-
-Era requires the Coordinator's UniqueID (or MRENCLAVE in SGX terms) or the tuple ProductID, SecurityVersion, SignerID (MRSIGNER) to verify the quote.
-In production, these would be generated when building the Coordinator and distributed to your clients.
-For testing, we have published a Coordinator image at `ghcr.io/edgelesssys/coordinator`.
+In production, the expected values in `coordinator-era.json` would be generated when building the Coordinator and distributed to your clients. When you build MarbleRun from source, you can find the file in your build directory.
+For testing with a pre-built release, we have published a Coordinator image at `ghcr.io/edgelesssys/coordinator`.
 You can pull the corresponding `coordinator-era.json` file from our release page:
 
 ```bash
 wget https://github.com/edgelesssys/marblerun/releases/latest/download/coordinator-era.json
 ```
 
+After installing Era, you can verify the quote with the following command:
```

Comment:
```suggestion
After installing era, you can verify the quote with the following command:
```

Reply:
It's capitalized before in the text. Change that one, too?

Reply:
Oh, I've missed that. Yes please.